leaderboard/controllers/index.js

```diff
@@ -0,0 +1,31 @@
+const Week = require('../models/weeks');
+
+module.exports = {
+
+	/* GET home page. */
+	getIndex (req, res, next) {
+	  res.render('index', { title: 'Express' });
+	},
+
+	/* GET about page. */
+	getAbout (req, res, next) {
+		res.render('about');
+	},
+
+	/* GET weeks page. */
+	async getWeeks (req, res, next) {
+		await Week.find({}, (err, weeks) => {

```

Comment:
no callback here, use `const weeks = await Week.find({});` and handle error with middleware or wrap in a try/catch block